React/Views/RCTPickerManager.m

```diff
@@ -19,7 +19,9 @@ @implementation RCTPickerManager
 
 - (UIView *)view
 {
-  return [RCTPicker new];
+  RCTPicker *picker = [RCTPicker new];
+  [picker selectRow:0 inComponent:0 animated:YES];
```

Comment:
It just should be inside RCTPicker's init method, and View Managers should be as declarative as possible.